common/lib/common-definitions/src/logger.ts

```diff
@@ -11,8 +11,17 @@ export type TelemetryEventCategory = "generic" | "error" | "performance";
 // General best practice is to explicitly log the fields you care about from objects
 export type TelemetryEventPropertyType = string | number | boolean | undefined;
 
+/**
+ * A property to be logged to telemetry containing both the value and a tag. Tags are generic strings that can be used
+ * to mark pieces of information that should be organized or handled differently by loggers in various first or third
```

Comment:
You might want to reference from here a place where FF tags are enumerated, but add that this list is not fixed (i.e. other tags can show up) because of dynamic nature of code (any FF runtime version can be loaded by host) and extensibility (3rd party), thus implementers of this interface should assume to see unknown tags.

Reply:
Currently this list is the 'TelemetryDataTag' enum in telemetry-utils/logger. This makes me think - since we're going to have a canonical set of internal tags, should I move the enum to common-definitions as well?

Reply:
Please note that even if we have canonical set of internal tags, this does not given consumers (host) any good guarantee, as driver, loader & runtime ship at different speeds, so they all could be using common-definitions of different version and thus have different set of tags emitted.
I do not have (yet) strong opinion on where to put them. There is benefits to both centralized and non-centralized approach

Reply:
I will keep them in `telemetry-utils` for now